weeks/week03/README.md

```diff
@@ -150,7 +150,7 @@ Unlike languages such as C, a JavaScript `Array` can contain any type of data, i
 var list = [0, "1", "two", true];
 ```
 
-JavaScript `Array`s can also contain holes (i.e., be missing certain elements), change size dynamically at runtime, and we don't need to specify an initial size:
+JavaScript `Arrays` can also contain holes (i.e., be missing certain elements), change size dynamically at runtime, and we don't need to specify an initial size:
```

Comment:
This is a common way to say the plural of a code term, e.g., the plural of `Array` is `Array`s, since `Arrays` doesn't exist as a term in the language.